lib/internal/v8_prof_polyfill.js

```diff
@@ -79,7 +79,7 @@ var line = '';
 
 function peekline() {
   const s = readline();
-  line = s + '\n' + line;
+  line = `s \n` + line;
```

Comment:
change `s` as `${s}` as we need to append the const variable s instead of the letter s.